Source/Core/Common/MemArena.h

```diff
@@ -112,15 +112,11 @@ class MemArena final
   void* m_address_UnmapViewOfFileEx = nullptr;
   void* m_address_VirtualAlloc2 = nullptr;
   void* m_address_MapViewOfFile3 = nullptr;
-#else
-#ifdef ANDROID
-  int fd;
 #else
   int m_shm_fd;
```

Comment:
A little bit unrelated, so feel free to ignore, but since this is being touched, should these also be initialized to something deterministic on construction?